linux_service/README.md

```diff
@@ -18,6 +18,8 @@ Three launch methods provided:
 3. Launch exporters and an [Azure Monitor](../docs/AzureMonitorAgent.md) publisher.
    - Before launch you must modify the "azure_monitor_agent_config" section of [publisher_config](../src/worker/publisher/config/publisher_config.json) file with the Azure Monitor workspace connection string.
 
+There is one additional method for internal Msft use that exports to Geneva. This method is similar to Azure Monitor method but uses a Geneva agent container to export. Reference the [Moneo Geneva Docs](../docs/GenevaAgent.MD). Ensure all prequisites are met. 
```

Comment:
Is this needed in the public documentation?

Reply:
Its easier to maintain the instructions here. All that is being shared is how to configure Moneo/VMSS.